runtime/onert/core/src/util/shapeinf/Permute.cc

```diff
@@ -56,16 +56,21 @@ void DynamicInferer::visit(const ir::operation::Permute &op)
   // getting output shapes
   auto new_shape = input_shape;
   // Permute is a special operation that layouts of input/output may be different
+  assert(new_shape.rank() <= ir::Shape::MAX_RANK);
   if (new_shape.rank() >= 4)
```

Comment:
should this be also changed to  the following? In case of rank 5, I don't think it is meaningful to convert NHWC NCHW conversion.
```suggestion
  if (new_shape.rank() == 4)
```

Reply:
I guess this can be committed in next PR.

Reply:
It is meaningful for the assertions to check `new_shape.rank() == 4`